addons/binding/org.openhab.binding.solarview/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Solarview Binding -->
+<thing:thing-descriptions bindingId="solarview"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Solarview energyProduction Binding -->
+    <thing-type id="energyProduction">
+        <label>Solarview Energy Production Information</label>
+        <description>Provides various energy data from the Solarview service</description>
+
+        <channels>
+            <channel id="KT0"	typeId="KT0"/>
```

Comment:
The names just came along from the Solarview monitoring (unfortunatelly only the [German documentation](http://www.solarview.info/solarview-fb_Installieren.pdf) is up-to-date).